### PR TITLE
revert to make accurate with lab

### PR DESCRIPTION
**Pull request by lmaxwell24**
includes:
- driver control changes to have blue origin at all times
- go from pigeon to navx
- changes tested on current swerve base for controls

### CODE REVIEW

File: commands/drive/absoluterelativedrive.py [hunk 1]
```diff
@@ -47,9 +51,18 @@ def rotation(self) -> float:
         )
 
     def execute(self) -> None:
-        self.drive.arcadeDriveWithFactors(
```
Comment (LandonBayer) [inline commands/drive/absoluterelativedrive.py:50]: Why did this get added back? I remember it getting removed specifically

File: robotcontainer.py [hunk 2]
```diff
@@ -149,15 +151,15 @@ def configureButtonBindings(self):
...
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Off)
-        )
-        ModifiableJoystickButton(self.operatorInterface.velocitySetpoint1).onTrue(
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Setpoint1)
-        )
-        ModifiableJoystickButton(self.operatorInterface.velocitySetpoint2).onTrue(
-            VelocitySetpoint(self.velocity, VelocityControl.ControlState.Setpoint2)
-        )
+        # ModifiableJoystickButton(self.operatorInterface.offVelocity).onTrue(
+        #     VelocitySetpoint(self.velocity, VelocityControl.ControlState.Off)
```
Comment (LandonBayer) [inline robotcontainer.py:155]: Why all commented out now

File: subsystems/drivesubsystem.py [hunk 3]
```diff
@@ -311,11 +310,7 @@ def __init__(self) -> None:
 
         # Create the gyro, a sensor which can indicate the heading of the robot relative
         # to a customizable position.
-        self.gyro = Pigeon2(constants.kPigeonCANId, constants.kCANivoreName)
-
-        toApply = Pigeon2Configuration()
-        self.gyro.configurator.apply(toApply)
-        self.gyro.get_yaw().set_update_frequency(100)
+        self.gyro = AHRS.create_spi()
```
Comment (LandonBayer) [inline subsystems/drivesubsystem.py:313]: Fine but we should fix when pigeons come in

File: commands/drive/absoluterelativedrive.py (hunk 1 above)
Comment (lmaxwell24) [reply commands/drive/absoluterelativedrive.py:50]: pathplanner 2024 has the same field symmetry issues of 2023 where you can't just assume red origin and blue origin are equal
this ensures that the robot is always in the perspective of blue origin

File: robotcontainer.py (hunk 2 above)
Comment (lmaxwell24) [reply robotcontainer.py:155]: velocity setpoint not relevant to the actual lab, putting it here as comment for as-needed with testing

File: subsystems/drivesubsystem.py (hunk 3 above)
Comment (lmaxwell24) [reply subsystems/drivesubsystem.py:313]: agreed, its a revertable commit

**Review by LandonBayer — APPROVED** [review]

**Review by icai0 — APPROVED** [review]

**Pull request merged**